mixer/template/tracespan/tracespan.proto

```diff
@@ -62,118 +62,86 @@ option (istio.mixer.adapter.model.v1beta1.template_variety) = TEMPLATE_VARIETY_R
-    // type.
-    //
-    // Optional
-    bool client_span = 9;
-
-    // rewrite_client_span_id is used to indicate whether to create a new client
-    // span id to accommodate Zipkin shared span model. Some tracing systems like
-    // Stackdriver separates a RPC into client span and server span. To solve this
-    // incompatibility, deterministically rewriting both span id of client span and
-    // parent span id of server span to the same newly generated id.
-    //
-    // Optional
-    bool rewrite_client_span_id = 10;
-
-    // Identifies the source (client side) of this span.
-    // Should usually be set to `source.workload.name`.
-    //
-    // Optional.
-    string source_name = 11;
-
-    // Client IP address. Should usually be set to `source.ip`.
-    //
-    // Optional.
-    istio.policy.v1beta1.IPAddress source_ip = 12;
-
-    // Identifies the destination (server side) of this span.
-    // Should usually be set to `destination.workload.name`.
-    //
-    // Optional.
-    string destination_name = 13;
-
-    // Server IP address. Should usually be set to `destination.ip`.
-    //
-    // Optional.
-    istio.policy.v1beta1.IPAddress destination_ip = 14;
-
-    // Request body size. Should usually be set to `request.size`.
-    //
-    // Optional.
-    int64 request_size = 15;
-
-    // Total request size (headers and body).
-    // Should usually be set to `request.total_size`.
-    //
-    // Optional.
-    int64 request_total_size = 16;
-
-    // Response body size. Should usually be set to `response.size`.
-    //
-    // Optional.
-    int64 response_size = 17;
-
-    // Response total size (headers and body).
-    // Should usually be set to `response.total_size`.
-    //
-    // Optional.
-    int64 response_total_size = 18;
-
-    // One of "http", "https", or "grpc" or any other value of
-    // the `api.protocol` attribute. Should usually be set to `api.protocol`.
-    //
-    // Optional.
-    string api_protocol = 19;
+  // Required. Trace ID is the unique identifier for a trace. All spans from the
+  // same trace share the same Trace ID.
+  string trace_id = 1;
+
+  // Optional. Span ID is the unique identifier for a span within a trace. It is
+  // assigned when the span is created.
+  string span_id = 2;
+
+  // Optional. Parent Span ID is the unique identifier for a parent span of this
+  // span instance. If this is a root span, then this field MUST be empty.
+  string parent_span_id = 3;
+
+  // Required. Span name is a description of the span's operation.
+  //
+  // For example, the name can be a qualified method name or a file name
+  // and a line number where the operation is called. A best practice is to use
+  // the same display name within an application and at the same call point.
+  // This makes it easier to correlate spans in different traces.
+  string span_name = 4;
+
+  // Required. The start time of the span.
+  // (-- api-linter: standard-fields=disabled --)
```

Comment:
i don't know that we need these api-linter instructions here. these probably aren't accounted for in the docs gen.